Tests/conf.json

```diff
@@ -2868,6 +2868,11 @@
         "Panorama Query Logs - Test": "Issue #23505",
         "palo_alto_panorama_test_pb": "Issue #22835",
         "VxStream Test": "Issue #23795",
+        "GCS Bucket Management - Test": "used in GCS - Test as sub playbook",
+        "GCS Object Operations - Test": "used in GCS - Test as sub playbook",
+        "GCS Bucket Policy (ACL) - Test": "used in GCS - Test as sub playbook",
+        "GCS Object Policy (ACL) - Test": "used in GCS - Test as sub playbook",
+        "Send Email To Recipients": "used in EWS Mail Sender Test 2 as sub playbook",
```

Comment:
this is spamming the list, can't we think of a better way? do we support `NonCircleTests` in packs `TestPlaybooks` folder?

Reply:
I agree this is a better solution, however it should in be another PR if we want this one to be merged soon